tools/system_libs.py

```diff
@@ -709,9 +716,9 @@ class libcxx(NoBCLibrary, CXXLibrary, NoExceptLibrary, MTLibrary):
     'variant.cpp',
     'vector.cpp',
     os.path.join('experimental', 'memory_resource.cpp'),
-    os.path.join('experimental', 'filesystem', 'directory_iterator.cpp'),
-    os.path.join('experimental', 'filesystem', 'path.cpp'),
```

Comment:
is filesystem/path.cpp no longer a thing?

Reply:
Seems not.